### DISCUSSION TITLE
Nested object in GET query params in RTK Query

### DISCUSSION THREAD

**Discussion by PttRulez** [reduxjs/redux-toolkit#3112] [Q&A]
Hey everyone.
I'm trying to make a GET request via RTK Query.
`{
    url: '/some-url',
    method: 'GET',
    params: { filters }
  }`

filters is an object. And it converts to "filters=%5Bobject+Object%5D" - what can I do about that? It worked perfectly fine when I used axios

**Comment by iuliuvisovan** [comment]
Any update on this?

**Comment by Dovakeidy** [comment]
RTK Query use **URLSearchParams** under the hood. You can customize this with the [**paramsSerializer**](https://redux-toolkit.js.org/rtk-query/api/fetchBaseQuery#paramsserializer) option of baseQuery. I personnaly use [qs ](https://github.com/ljharb/qs) instead.

Eg. 

```ts
const baseQuery = fetchBaseQuery({
  baseUrl: API_URL,
  paramsSerializer: (params) => {
    return qs.stringify(params);
  }
})
```
